packet/templates/active_packets.html

```diff
@@ -25,41 +25,39 @@ <h3 class="page-title">Active Packets</h3>
                                         </thead>
                                         <tbody>
                                         {% for packet in packets %}
-                                            {% if packet.is_open() %}
-                                                <tr {% if packet.did_sign %}style="background-color: #4caf505e" {% endif %}>
-                                                    <td>
-                                                        <a href="/packet/{{ packet.freshman.rit_username }}">
-                                                            <img class="eval-user-img"
-                                                                 alt="{{ packet.freshman.name }}"
-                                                                 src="https://www.gravatar.com/avatar/freshmen?d=mp&f=y"
-                                                                 width="25"
-                                                                 height="25"/> {{ packet.freshman.name }} ({{ packet.freshman.rit_username }})
-                                                        </a>
-                                                    </td>
-                                                    <td>
-                                                        {% if packet.total_signatures == packet.required_signatures %}
-                                                            💯 {# 100% emoji #}
+                                            <tr {% if packet.did_sign %}style="background-color: #4caf505e" {% endif %}>
```

Comment:
There is a merge conflict in this file. Can you rebase your changes to upstream?

`git fetch upstream develop`
`git rebase upstream/develop`
Fix conflicts
`git rebase --continue`
`git push -f`